Bookish/Models/SearchBooksModel.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Bookish.Models;
 
-public class SearchModel
+public class SearchBooksModel

```

Comment:
Similarly to earlier, I don't think this is really a "model" - it's a helper class filled with helper methods. I'd rename and put in a Helpers folder